install_phantomjs.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash
-export PHANTOM_JS="phantomjs-1.9.8-linux-x86_64"
-export DEST="/usr/local/share"
 
-set -e
+phantom_versions=(phantomjs-1.9.8-linux-x86_64 phantomjs-2.1.1-linux-x86_64)
 
-apt-get install -y build-essential chrpath libssl-dev libxft-dev libfreetype6 libfreetype6-dev libfontconfig1 libfontconfig1-dev wget
-mkdir -p $DEST
-wget -O - -q https://bitbucket.org/ariya/phantomjs/downloads/$PHANTOM_JS.tar.bz2 | tar -xvjC $DEST
-ln -s ${DEST}/${PHANTOM_JS}/bin/phantomjs /usr/local/bin/
+for phantom_version in ${phantom_versions[@]}; do
+    export PHANTOM_JS=${phantom_version}
+    export DEST="/usr/local/share"
+
+    set -e
+
+    apt-get install -y build-essential chrpath libssl-dev libxft-dev libfreetype6 libfreetype6-dev libfontconfig1 libfontconfig1-dev wget
+    mkdir -p $DEST
+    wget -O - -q https://bitbucket.org/ariya/phantomjs/downloads/$PHANTOM_JS.tar.bz2 | tar -xvjC $DEST
+    ln -s ${DEST}/${PHANTOM_JS}/bin/phantomjs /usr/local/bin/
```

Comment:
Do we need the link?  If that is being used by tests this might not work.  We could alternatively create different containers with different phantom versions and label it and require that label in avvo-merges